src/components/Header/HeaderAvatar.jsx

```diff
@@ -42,6 +62,8 @@ HeaderAvatar.propTypes = {
   shortName: PropTypes.string.isRequired,
   /** The URL for the logout action */
   logoutURL: PropTypes.string,
+  /** The onClick for the logout action*/
```

Comment:
The docs should probably indicate that `logoutClick` and `logoutUrl` are mutually exclusive:

```suggestion
  /** The onClick for the logout action. This value will be overriden by logoutUrl if it is provided  */
```